tiledb/libtiledb.pyx

```diff
@@ -5201,6 +5227,9 @@ def _setitem_impl_sparse(self: Array, selection, val, dict nullmaps):
                                      "typed attribute '{}'!".format(name))
 
                 attr_val = np.ascontiguousarray(attr_val, dtype=attr.dtype)
+            
+            if attr.isnullable and attr.name not in nullmaps:
+                nullmaps[attr.name] = np.array([int(v is None) for v in attr_val], dtype=np.uint8)
```

Comment:
Why do we need to do this? (also, calling `int(v is None) for v in attr_val` seems like it will be really really slow...)